Resources/Textures/DeltaV/Objects/Misc/bureaucracy.rsi/meta.json

```diff
@@ -1,14 +1,14 @@
 {
```

Comment:
untouch this file its unrelated

Reply:
The stamp was there previously until i learned it's hardcoded to check on the upstream files.